src/components/NewCommentForm.tsx

```diff
@@ -59,13 +76,10 @@ export const NewCommentForm: React.FC<Props> = ({ onSubmit }) => {
 
     setSubmitting(true);
 
-    // it is very easy to forget about `await` keyword
-    await onSubmit({ name, email, body });
+    await createNewComment({ name, email, body });
```

Comment:
You are not handling possible errors that could occur when creating a new comment. Consider wrapping this call in a try-catch block to handle any errors that might occur and provide feedback to the user.